app/helpers/application_helper.rb

```diff
@@ -18,13 +18,15 @@ def group_navigation_presenter
   #  - fallback to Hyrax's default image
   def collection_thumbnail(document, _image_options = {}, url_options = {})
     view_class = url_options[:class]
-    # The correct thumbnail SHOULD be indexed on the object
-    return image_tag(document['thumbnail_path_ss'], class: view_class, alt: alttext_for(document)) if document['thumbnail_path_ss'].present?
+    # Use collection branding thumbnail if it exists
+    if document['thumbnail_path_ss'].present? && !document['thumbnail_path_ss'].include?('default_work_images')
```

Comment:
This indicates that the wrong thumbnail is indexed... should we do something about this situation? I don't know the answer, but it seems like we'd want to remedy it somehow...

Reply:
@laritakr yes, that appears to be what's happening. it gets the work default image because of[ this line](https://github.com/samvera/hyku/blob/main/app/services/hyrax/thumbnail_path_service_decorator.rb#L13). 

And you're right, instead of a band aid let me try to prevent it from doing that for collections.

Reply:
Actually, I think all i need to do is check if it's a collection and called #default_collection_image instead 🤔